src/component/Home.js

```diff
@@ -11,7 +11,7 @@ import heroImage from "../style/images/homepage-hero.jpg";
  * The default component of pmo.
  * @returns {JSX.Element}
  */
-export default () => {
+const Home = () => {
```

Comment:
I generally used named functions here instead of arrow functions. Can you confirm that these names are picked up by React dev tools?

Reply:
Yup - React tools sees names.

Reply:
I know it sees function names, I didn't think it would see variable names. Is it accessible in `Home.name === 'Home'`? A lot of canjs debug tools (which ylem uses) rely on this.